### PR TITLE
Added functionality to read excel without saving

### PR DESCRIPTION
**Pull request by twitu**
updated requirements also requires openpyxl

### CODE REVIEW

**Comment by rtgdk** [comment]
@Ishan-sopho  You forgot to add models for the menu. Also change the name of the view function from `index` to `menu`. And make a separate page and url for students to view the menu and admin to upload the menu("/ssms/menu_upload/")

**Comment by rtgdk** [comment]
Also test whether the whole thing is working or not.

File: ssms/views.py [hunk 1]
```diff
@@ -1151,13 +1151,52 @@ def student_grub_feedback(request,gmid):
...
-			students = Grub_Student.objects.filter().order_by('bhawan')
-			count = len(students)
-		else :
-			return render(request,'ssms/menu.html')
+def index(request):
+    # check if request is post and handle appropriately by storing file in media
+    if request.method == 'POST' and request.FILES['myfile']:
+        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'dailymenu.settings')
+        django.setup()
+        # file path can be filled manually or using file url generated by file upload.
```
Comment (rtgdk) [inline ssms/views.py:1160]: There is no app named dailymenu. You can remove the last two lines. Django uses its default setup.

File: ssms/views.py [hunk 2]
```diff
@@ -1151,13 +1151,52 @@ def student_grub_feedback(request,gmid):
...
 			
-def menu(request):
-	if request.user.is_superuser:
-		grub = Grub.objects.get(gm_id=gmid)
-		if request.method == 'POST':
-			students = Grub_Student.objects.filter().order_by('bhawan')
-			count = len(students)
-		else :
-			return render(request,'ssms/menu.html')
+def index(request):
```
Comment (rtgdk) [inline ssms/views.py]: Change the function name to menu

**Comment by twitu** [comment]
this should work i have tested it locally but on a separate django project

**Comment by rtgdk** [comment]
Excellent work @Ishan-sopho  ! :+1: 
Just a few changes. You forgot to import Meal and Items in views. Also remove these two lines `os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'dailymenu.settings')` , `django.setup()`.

**Comment by rtgdk** [comment]
Read the reviews and changes accordingly.

File: ssms/urls.py [hunk 3]
```diff
@@ -44,5 +44,6 @@
 	url('^ajax/ssms/grub/(?P<gmid>[-\w]+)/sendmail2/' , views.ssms_grub_sendmail2, name="Grub send mail1") ,
 	#url(r'^(?P<inv>\w+)/$',views.invalid,name="invalid"),
 	#url(r'^coord/upload-spot-signing/(?P<gmid>[-\w]+)/$', views.coord_upload_spot, name='upload spot signing'),
+	url(r'^/ssms/menu_upload/$', views.menu_upload, name='menu_upload'),
+	url(r'^/ssms/menu/$', views.menu_display, name='menu_display'),
```
Comment (rtgdk) [inline ssms/urls.py:48]: It should be just `ssms/menu`. No "/" needed before ssms. Same in menu_upload.

**Pull request merged**